static/js/tools/download/info.tsx

```diff
@@ -20,7 +20,20 @@
 
 import React from "react";
 
-export function Info(): JSX.Element {
+// Data object used to customize the example rendered by this component.
+// A single example place to display.
+export interface InfoPlace {
+  name: string;
+  dcid: string;
+}
+
+// Data object used to customize the example rendered by this component.
+interface InfoPropType {
+  // NOTE: Exactly 2 InfoPlace objects are expected.
+  infoPlaces: InfoPlace[];
```

Comment:
Wonder can do [InfoPlace, InfoPlace] here to indicate 2 elements?

Reply:
Ah, that works! Thanks for the suggestion!